### ISSUE TITLE
Show alarm in date/time widget

### ISSUE THREAD

**Issue by 8bitPit** [8bitPit/Niagara-Issues#4]

**Comment by 587saboor** [comment]
There should be an option to turn it on or off because some people like it without the alarm clock

**Comment by 8bitPit** [comment]
Last days I reflected about this feature and came to the conclusion that  this is feature is not really necessary. A little alarm icon on the top right already indicates that an alarm has been set. If you open Android's notification shade you can also see the time of the alarm. In contrast to alarms, reminders and appointments don't get displayed here. I would like to show those on the home screen instead: #55

**Comment by pratyakshm** [comment]
I agree with what @8bitPit said in https://github.com/8bitPit/Niagara-Issues/issues/4#issuecomment-569490201, however my setup of Niagara has Status bar hidden. I did this because Niagara provides all the information that status bar provided (in my personal use case). I would appreciate if the same happens with alarms. Please try re-considering this if possible. If you still don't intend to add the alarm status, its fine. Thanks for taking your time to read this.

**Comment by saboooor** [comment]
:o

**Comment by JanPokorny** [comment]
@8bitPit My phone (Android 12, with a screen notch) does not show the alarm time anywhere in the notification dropdown, and the alarm icon is not visible in the statusbar unless the notification shade is pulled down. I think that showing the alarm time in the same way the vanilla Android lockscreen does would be beneficial.

**Comment by amalsk7** [comment]
So...any way to implement this an as option for those who want it? Keeping a toggle for those who want this feature would be great! I really wish to see my next alarm when looking at my clock widget, like I was previously able to do, but on the Niagara launcher's clock itself.

**Comment by 8bitPit** [comment]
I'm sorry, we ranked showing the alarm clock next to the clock a low-priority feature, so we can't guarantee when and if we implement it.

**Comment by 8bitPit** [comment]
As a workaround, I recommend adding a custom clock widget (e.g. from Google's clock app) and replace Niagara Launcher's clock with it (long press the widget > Replace clock with widget).
